apps/src/util/openai.js

```diff
@@ -0,0 +1,13 @@
+export const openaiCompletion = messages => {
```

Comment:
I might just drop this file from this PR and focus on the Rails side, and add this when we have a usage for it?

If you decide to include this, maybe worthwhile to use this handy utility Sanchit put together to make sure you have a valid CSRF token:

https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/AuthenticityTokenStore.ts

Reply:
I'll go ahead and remove for now -thanks!